composable_kernel/include/tensor_operation/reduction_functions_blockwise.hpp

```diff
@@ -23,247 +23,161 @@
  * SOFTWARE.
  *
  *******************************************************************************/
-#ifndef CK_REDUCTION_FUNCTIONS_BLOCKWISE_HPP
-#define CK_REDUCTION_FUNCTIONS_BLOCKWISE_HPP
+#ifndef CK_REDUCTION_FUNCTIONS_PARTITIONED_BLOCKWISE_HPP
+#define CK_REDUCTION_FUNCTIONS_PARTITIONED_BLOCKWISE_HPP
 
 #include "data_type.hpp"
 
 #include "reduction_common.hpp"
 #include "reduction_operator.hpp"
-#include "reduction_functions_binop.hpp"
+#include "reduction_functions_accumulate.hpp"
 
 namespace ck {
 
-template <typename buffer2dDescType,
-          bool blockIsOneRow,
+template <typename buffer1dDescType,
+          typename AccDataType,
+          index_t BlockSize,
+          index_t MThreadClusterSize,
+          index_t KThreadClusterSize,
```

Comment:
typename ThreadClusterLengths_M_K

Reply:
Will change to use 2D descriptor next time